enodi/PostIt-RoR-
# #156947369 Enable user create a new group

#### What does this PR do?
This PR handles creating a new group

#### Description of Task to be completed?
- Creates group model/controller
- Validate input params
- Enable user to send a post request to create a new group

#### How should this be manually tested?
- Pull this branch and run migration i.e `rails db:migrate`
- Run `rails s`
- To make a request to the API, use an HTTP client testing tool e.g `postman`.
- To test the group route on `postman`, send a post request with valid credentials i.e `name` and `description` to the URL: `api/v1/group`.
- The Authorization token should be placed in the `headers`.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[Story ID: 156947369](https://www.pivotaltracker.com/n/projects/2166405/stories/156947369)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29783644/39062932-2bd32000-44c1-11e8-9428-9b7cc2813380.png)
